dask/array/fft.py

```diff
@@ -86,19 +89,73 @@ def _ihfft_out_chunks(a, n, axis):
     return chunks
 
 
-fft = _fft_wrap(npfft.fft, np.complex_, _fft_out_chunks)
+def fft_wrapper(fft_func):
+    """ Wrapper of 1D complex FFT functions
+
+    Takes a function that behaves like ``numpy.fft.fft``.
+    """
+
+    return _fft_wrap(np.complex_, _fft_out_chunks)(fft_func)
+
+
+def ifft_wrapper(fft_func):
+    """ wrapper of 1D complex inverse FFT functions
+
+    takes a function that behaves like ``numpy.fft.ifft``.
+    """
+
+    return _fft_wrap(np.complex_, _fft_out_chunks)(fft_func)
+
+
+def rfft_wrapper(fft_func):
+    """ wrapper of 1D real FFT functions
+
+    takes a function that behaves like ``numpy.fft.rfft``.
+    """
+
+    return _fft_wrap(np.complex_, _rfft_out_chunks)(fft_func)
+
+
+def irfft_wrapper(fft_func):
+    """ wrapper of 1D real inverse FFT functions
+
+    takes a function that behaves like ``numpy.fft.irfft``.
+    """
+
+    return _fft_wrap(np.float_, _irfft_out_chunks)(fft_func)
+
+
+def hfft_wrapper(fft_func):
+    """ wrapper of 1D hermitian complex FFT functions
+
+    takes a function that behaves like ``numpy.fft.hfft``.
+    """
+
+    return _fft_wrap(np.float_, _hfft_out_chunks)(fft_func)
+
+
+def ihfft_wrapper(fft_func):
+    """ wrapper of 1D hermitian complex inverse FFT functions
+
+    takes a function that behaves like ``numpy.fft.ihfft``.
+    """
+
+    return _fft_wrap(np.complex_, _ihfft_out_chunks)(fft_func)
```

Comment:
Well, I'm planning to add the 2-D and N-D versions of these too. So as long as we don't think that this will make the function too long with such a big branch that is fine. Though some linting tools may balk at this.